app/scripts/modules/core/src/account/AccountSelectField.tsx

```diff
@@ -1,13 +1,142 @@
+import * as React from 'react';
+import { $q } from 'ngimport';
+import { flatten, has, intersection, isEqual, map, uniq, xor } from 'lodash';
+
 import { IAccount } from 'core/account';
+import { AccountService } from 'core/account/AccountService';
 
 export interface IAccountSelectFieldProps {
   accounts: IAccount[] | string[];
-  component: Object;
+  component: { [key: string]: any };
```

Comment:
I think you could use `object` with lower case `o` if you prefer

Reply:
Oh nice, didn't know about that one.

Reply:
Ah, then can't do `object[key]` because it complains of no index signature. sad face.